pkgs/development/python-modules/pytest-helpers-namespace/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, pytest
+, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-helpers-namespace";
+  version = "2019.1.8";
+
+  src = fetchFromGitHub {
+    owner = "saltstack";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "0z9f25d2wpf3lnqzmmnrlvl5b1f7kqwjjf4nzs9x2bpf91s5zny1";
+  };
+
+  propagatedBuildInputs = [ pytest ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
Added a comment. It's due to conflicting pytest versions between what works for this and what is required for pikepdf to work.